### PR TITLE
Ignore GitHub's `pull_request_review_comment` on edited

### PR DESCRIPTION
**Pull request by tricknotes**
TODO:
* [x] It works
* [x] Add test

### CODE REVIEW

File: lib/hooks/github/hook.rb [hunk 1]
```diff
@@ -51,7 +51,7 @@ def pending_status_event?
     end
 
     def edit_or_delete_issue_comment_event?
-      event_type == 'issue_comment' && %w(edited deleted).include?(payload.action)
+      %w(issue_comment pull_request_review_comment).include?(event_type) && %w(edited deleted).include?(payload.action)
```
Comment (tricknotes) [inline lib/hooks/github/hook.rb]: This is a test comment.

**Comment by tricknotes** [comment]
We can ignore GitHub's edited `pull_request_review_comment` event in favor of https://github.com/idobata/idobata-hooks/pull/67 .

File: lib/hooks/github/hook.rb [hunk 2]
```diff
@@ -51,7 +51,7 @@ def pending_status_event?
     end
 
     def edit_or_delete_issue_comment_event?
```
Comment (ursm) [inline lib/hooks/github/hook.rb]: How about changing this to `edit_or_delete_comment_event?`?

Comment (tricknotes) [inline lib/hooks/github/hook.rb]: Sound good!
Now I updated my commit. 434f4c66abb96200ea9a39475ce94d25666a5890

**Comment by ursm** [comment]
Awesome!

**Pull request merged**